ReviewSite/lib/js/scripts/home.js

```diff
@@ -1,11 +1,21 @@
-import {getReviewList, likeReview, deleteReview, isLiked} from "./data/data_managment.js";
-import {isCreator} from "../app-modules/node-postgres/review_queries";
+import {getReviewList, likeReview, deleteReview, isLiked} from "./data_managment.js";
 
-window.addEventListener('load', async () => {
-    await listReviews(await getReviewList());
-});
+export async function loadHomePage(navigateTo){
+    await listReviews(await getReviewList(), navigateTo);
 
-async function listReviews(reviews) {
```

Comment:
here, it would be better to make a component, or a view in ur case, smth like 'list-item', and a view 'list' and than, after downloading ur home page make a call to retrieve all ur records from db, like u are doing rn. and then render ur [list](https://marionette.gitbooks.io/marionette-guides/content/en/views/collections.html) with passing ur information there